src/main/java/com/dawn/jat/illuminati/post/controller/PostController.java

```diff
@@ -38,4 +37,23 @@
         }
         return new ResponseEntity<>(postEntity, HttpStatus.OK);
     }
-}
+
+    /**
+     * Gets list of Post Summary.
+     * @throws PostSummaryNotFoundException if Post Summary has No Content
+     */
+    @GetMapping(value = "/all-post/summary")
+    public ResponseEntity<Object> getAllPostSummary() {
+        List<PostSummaryEntity> allPost = postService.findPostSummary();
+
+        Map<String, Object> body = new LinkedHashMap<>();
+
+        if (allPost.isEmpty()) {
+            throw new PostSummaryNotFoundException("Cannot find post summary");
+        }
+
+        body.put("data", allPost);
+        body.put("success", true);
```

Comment:
We should create 2 objects SuccessResponse as oppose to ErrorResponse for this

Reply:
Updated

Reply:
![image](https://user-images.githubusercontent.com/7389488/72913485-c7bdf280-3d6f-11ea-9298-1c9ade390aa7.png)